src/prefect/core/flow.py

```diff
@@ -805,6 +806,24 @@ def edges_from(self, task: Task) -> Set[Edge]:
             )
         return self.all_downstream_edges()[task]
 
+    def is_mapped(self, task: Task) -> bool:
+        """
+        Checks if task will map over at least one of its upstream task outputs
+
+        Args:
+            - task (Task): The task that we want to know if it is mapped
+
+        Returns:
+            - bool: `True` if the Task is mapped in this flow, `False` otherwise
+
+        Raises:
+            - ValueError: if `task` is not found in this flow
+        """
+        for edge in filter(lambda e: e.mapped, self.edges_to(task)):
+            return True
+
+        return False
```

Comment:
I prefer something like rather than a `filter`/`lambda`
```suggestion
        return any(edge.mapped for edge in self.edges_to(task))
```